app/models/server_role.rb

```diff
@@ -5,32 +5,19 @@ class ServerRole < ActiveRecord::Base
   validates_presence_of     :name
   validates_uniqueness_of   :name
 
-  FIXTURE_DIR = File.join(Rails.root, "db/fixtures")
-
-  def self.seed_data
-    File.read(File.join(FIXTURE_DIR, "#{to_s.pluralize.underscore}.csv"))
-  end
-
   def self.seed
-    data = seed_data.split("\n")
-    cols = data.shift.split(",")
-
-    data.each do |a|
-      next if a =~ /^#.*$/ # skip commented lines
+    CSV.foreach(fixture_path, :headers => true, :skip_lines => /^#/).each do |csv_row|
+      action = csv_row.to_hash
+      action['license_required'] ||= ''
```

Comment:
I've grepped through the code and haven't found any usage of `license_required `, except in tests, initial schema migration and in the line we're talking about here. Strictly speaking, there still might be a chance that it does matter whether it is `nil` or `""` in this column, but I'm hoping that this is not the case. I'm going to go ahead and delete this line of code and update the test. @Fryguy what do you think?